Remotion/Data/DomainObjects.UnitTests/ReflectionUtilityTest.cs

```diff
@@ -83,12 +80,21 @@ public void GetAssemblyPath_WithHashInDirectoryName ()
             },
             directoryPath,
             newAssemblyPath);
+#else
+        using var assemblyContext = new UnloadableAssemblyContext();
+        assemblyContext.RunWithAssemblyLoadContext(
+            context =>
+            {
+              var assembly = context.LoadFromAssemblyPath(newAssemblyPath);
+              Assert.That(Path.GetDirectoryName(assembly.Location), Is.EqualTo(directoryPath));
+              Assert.That(ReflectionUtility.GetAssemblyDirectory(assembly), Is.EqualTo(Path.TrimEndingDirectorySeparator(AppContext.BaseDirectory)));
```

Comment:
It feels like we should de-duplicate the logic in the test. It's just the knowledge that .NET Framework will die in less than a year that I'm ambivalent about changing it now.

Reply:
In this case there are differences between .NET Framework and .NET Core so I would not try to deduplicate the logic here. I think that the de-duplication would add more complexity than the conditional. We don't have many places where we use conditional code.

Reply:
Ah, the assembly-load is also different. hmm... okay.